ci-operator/step-registry/fleetshift/ci/image-mirror/fleetshift-ci-image-mirror-commands.sh

```
@@ -0,0 +1,78 @@
+
+# Build destination image reference
+DESTINATION_IMAGE_REF="$REGISTRY_HOST/$REGISTRY_ORG/$IMAGE_REPO:$IMAGE_TAG"
+
+log "INFO Mirroring Image"
+log "     From: $SOURCE_IMAGE_REF"
+log "     To  : $DESTINATION_IMAGE_REF"
+
+mirror_log="${ARTIFACT_DIR}/oc-mirror-output.log"
+
+for i in {1..6}; do
+    if ! oc image mirror --keep-manifest-list=true "$SOURCE_IMAGE_REF" "$DESTINATION_IMAGE_REF" 1>${mirror_log}; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)